legacy/plugin.ts

```diff
@@ -100,6 +100,9 @@ export interface PluginMetadata {
 export interface SinglePackageResult {
   plugin: PluginMetadata;
   package: DepTree;
+  meta?: {
+    gradleProjectName?: any,
```

Comment:
why not `string`? it is always a string right?

Reply:
good idea.
will change it to `string`